source/refined-github.css

```diff
@@ -189,6 +189,10 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 	border-bottom-left-radius: 0;
 }
 
+.flex-grow-1 {
+	flex-grow: 1 !important;
```

Comment:
I guess the `.flex-1` or `.flex-auto` classes in Primer (https://primer.style/css/utilities/flexbox#flex) are not sufficient? See also https://developer.mozilla.org/en-US/docs/Web/CSS/flex#formal_definition.